modules/accounts/accounts.js

```diff
@@ -1,191 +1,250 @@
-            return Helper.output(Helper.formatTez(r/100));
-        }).catch(function(e){
-            return Helper.outputError(e);
+
+        const keys = this.getKeys(account);
+        if(!keys) {
+            return Logger.error(`Account with this label doesn't exists.`);
+        }
+
+        await RpcRequest.fetchBalance(tezosNode, pkh).then(function(body){
+            var balance = JSON.parse(body);
+            Logger.info(Helper.formatTez(balance));    
+        }).catch(function (error) {
+            Logger.error(`${error}`);
         });
     }
 
-    createTestnetAccount(args){
-        var pkh = args[0], f;  
-        if (Helper.findKeyObj(config.accounts, args[1])) 
-        return console.log(Helper.outputError('That account name is already in use'));
-        if (f = Helper.findKeyObj(config.identities, pkh)) {
-            return eztz.rpc.account(f, parseFloat(args[2]), true, true,f.pkh, 1400).then(function(r) {                  
-                var d=eztz.contract.hash(r.hash, 0);        
-                config.accounts.push({
-                    label : args[1],
-                    pkh : d,
-                    identity : pkh,        
-                });
-                jsonfile.writeFile(confFile, config);
-                return Helper.output('New account created ' + args[1]);
-            }).catch(function(e) {           
-                return Helper.outputError(e);
-            });
-        } else {
-            return Helper.outputError(pkh + ' is not a valid identity');
+    async createTestnetAccount(args) {
+        var accountLabel = args[0];
+        const conseiljs = require(CONSEIL_JS);
+        const keys = this.getKeys(accountLabel);
+        if(keys) {
+            return Logger.error(`Account with this label already exists.`);
         }
-    }
 
-    async loadTezsterConfig() {
-        eztz = require('../../lib/eztz.cli.js').eztz;
-        const jsonfile = require('jsonfile');
-        config = jsonfile.readFileSync(confFile);
-        if (config.provider) {
-            eztz.node.setProvider(config.provider);
-        }  
-        const _sodium = require('libsodium-wrappers');
-        await _sodium.ready;
-        eztz.library.sodium = _sodium;
+        try {
+            const mnemonic = conseiljs.TezosWalletUtil.generateMnemonic();
+            const keystore = await conseiljs.TezosWalletUtil.unlockIdentityWithMnemonic(mnemonic, '');
+            this.addIdentity(accountLabel, keystore.privateKey, keystore.publicKey, keystore.publicKeyHash, '');
+            this.addAccount(accountLabel, keystore.publicKeyHash, accountLabel, config.provider);     
+            jsonfile.writeFile(confFile, config);
+            Logger.info(`Successfully created wallet with label: '${accountLabel}' and public key hash: '${keystore.publicKeyHash}'`);
+            return Logger.warn(`We suggest you to store following Mnemonic Pharase which can be used to restore wallet in case you lost wallet:\n'${mnemonic}'`);
```

Comment:
why do you return logger.warn()? pls fix this in other place as well. 
you method should have a definite return type, if you don't have anything to return, it'll return undefined by default.